### PR TITLE
chore: revamp GH issues crossroad and templates

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

Issues crossroads UI:
image

- 1 template for bug reports
  - report enforced to select "framework" for which they are reporting to issue 
- 1 template for feature request
- blank issues are enabled

## New Behavior

> [Preview templates 👁️](https://github.com/microsoft/fluentui/tree/8f7585c058daf4ed56b8966e28d03e699f2c0f89/.github/ISSUE_TEMPLATE)

- 5 templates for bug reports
  - react-components / [preview](https://github.com/microsoft/fluentui/blob/8f7585c058daf4ed56b8966e28d03e699f2c0f89/.github/ISSUE_TEMPLATE/01-react-components-bug-report.yml) 
  - react v8
  - react-charting
  - web-components
  - tooling
- 1 template for Documentation related issues/features / [preview](https://github.com/microsoft/fluentui/blob/8f7585c058daf4ed56b8966e28d03e699f2c0f89/.github/ISSUE_TEMPLATE/06-documentation.yml)
- 1 template for feature request
  - feature support only for 4 repo areas / [preview](https://github.com/microsoft/fluentui/blob/8f7585c058daf4ed56b8966e28d03e699f2c0f89/.github/ISSUE_TEMPLATE/07-feature-request.yml)
    - react-components
    - react-charting
    - web-components
    - tooling  
- blank issues are disabled 

### Library version fields

**Before:**

Not Present

**After:**

image

### Improved Environment reporting

**Before:**

image

**After:**

image

### Severity 

**Before:**

image

**After:**

image

### Reproduction

**Before:**

image

**After:**

image

### Splitting Description in 2

**Before:**

image

**After:**

image

## Related Issue(s)

- Implements partially #25697

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by AtishayMsft — APPROVED** [review]
Thanks for a charting specific bug template Martin.

**Review by khmakoto — APPROVED** [review]

**Review by JustSlone — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/01-react-components-bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    validations:
+      required: true
+
+  - type: input
+    id: reproduction-link
+    attributes:
+      label: Reproduction
+      description: |
+        A link to a Stackblitz reproduction which demonstrates the bug - https://stackblitz.com/
+
```
Comment (layershifter) [inline .github/ISSUE_TEMPLATE/01-react-components-bug-report.yml:136]:
```suggestion
> 💡 You can export to Stackblitz any example from docs (https://react.fluentui.dev/) with "Open In StackBlitz" button and update it to reproduce an issue.
```

Let's mention this as creating an example from scratch requires a _bit_ of boilerplate.

**Review by layershifter — APPROVED** [review]

**Comment by Hotell** [comment]
> image
> Didn't we agree to remove these?

that was part of proposal ( optional ), we can remove as a follow-up

**Review by tudorpopams — APPROVED** [review]

**Comment by dmytrokirpa** [comment]
I'd like to add a small suggestion regarding the organization of our bug report templates. For better readability, perhaps we could arrange the sections in this order:

Summary
...
Steps to Reproduce
Actual Behavior
Expected Behavior
...

instead of:
...
Actual Behavior
Expected Behavior
Steps to Reproduce
...

**Comment by Hotell** [comment]
> I'd like to add a small suggestion regarding the organization of our bug report templates. For better readability, perhaps we could arrange the sections in this order:
> 
> * ...
> * Summary
> * Steps to Reproduce
> * Actual Behavior
> * Expected Behavior
> * ...
> 
> instead of:
> 
> * ...
> * Actual Behavior
> * Expected Behavior
> * Steps to Reproduce
> * ...


**Adding summary** would make actual behaviour/expected behaviour kind ambiguos.  The intent here is to have things focused. We had "summary" before which contained "actual/expected" chapter which wasn't providing much clarity into what is actually needed from the reporter.

image

**Actual/Expected <-> Steps to Reproduce**

Initially I wanted to do it like this, I took data from various popular OSS repos and there is no clear pattern, some use it like in this PR, some like you suggest. One "against" having "steps to repro first" is the flow of person triaging. It doesn't make much sense to read Steps first when you don't know what those steps should demonstrate


For now I suggest to keep it as is, but ofc happy to tweak things as we go.

**Pull request merged**